src/transformStateWithClones.js

```diff
@@ -7,7 +7,57 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const NewArray = [];
+  let massive = { ...state };
```

Comment:
Better use more informative names for variables, 'massive' means 'large' in English, you probably meant 'array' or something like that🙃